src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCountsIntegrationTest.java

```diff
@@ -374,8 +348,8 @@ private RealMatrix readBetaHats(File betaHatsOutput, ReadCountCollection input)
         return new Array2DRowRealMatrix(betaHats,false);
     }
 
-    @Test(expectedExceptions = UserException.BadInput.class)
```

Comment:
Same thing: despite the name, it was only meant to fail due to the multiple samples, not the `CoordOnly`-ness.